tests/ios/unit-test/test-imports-app/frameworks/Nested/BUILD.bazel

```diff
@@ -1,7 +1,7 @@
 load("//rules:framework.bzl", "apple_framework")
 
 apple_framework(
-    name = "NestedHeaders",
```

Comment:
The nice thing here - detecting `Headers` captures and tests an obscure usage and some of bugs in rules_ios like
https://github.com/bazelbuild/rules_apple/pull/1798/files 🙈 

Is there any way we can undo this now that you've got it fixed in rules_apple. Also I wonder why `objc_import` is even dealing with headers now - I think there was a limitation around it that caused it to only pull linkable files like `.a` or `.so`

Reply:
Yeah, since I've gotten it fixed in `rules_apple`, I'm happy to revert this change and update to an even newer `rules_apple` commit.